spec.html

```diff
@@ -426,16 +434,12 @@ <h1>Runtime Semantics: ClassElementEvaluation</h1>
       1. Let _scope_ be the running execution context's LexicalEnvironment.
       1. Let _closure_ be FunctionCreate(~Method~, |PropertySetParameterList|, |FunctionBody|, _scope_, _strict_).
       1. Perform MakeMethod(_closure_, _homeObject_).
-      1. Perform SetFunctionName(_closure_, _key_, `"set"`).
+      1. Perform ! SetFunctionName(_closure_, _key_, `"set"`).
       1. <del>Let _desc_ be the PropertyDescriptor{[[Set]]: _closure_, [[Enumerable]]: _enumerable_, [[Configurable]]: *true*}.</del>
       1. <del>Return ? DefinePropertyOrThrow(_homeObject_, _propKey_, _desc_).</del>
+      1. <ins>Let _desc_ be ! DefaultMethodPropertyDescriptor(_key_, _closure_, _enumerable_).</ins>
```

Comment:
It's not valid to call DefaultMethodPropertyDescriptor here, since that abstract operation adds closure as `[[Value]]`, but we need to add it as `[[Set]]`.